### PR TITLE
vradarserver.vrs version 2.4.4

### PR DESCRIPTION
**Pull request by Boxersteavee**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26885)

### CODE REVIEW

**Comment by Boxersteavee** [comment]
What happened there.... It worked fine for me during testing

File: manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+LicenseUrl: https://github.com/vradarserver/vrs/blob/master/LICENSE.md
+CopyrightUrl: https://github.com/vradarserver/vrs/blob/master/LICENSE.md
+ShortDescription: Virtual Radar Setup
+Description: '"Virtual Radar Server is an open-source .NET application that runs a local web server"'
+Moniker: Virtual Radar
+Tags:
+- vrs
+- Virtual Radar
+- Virtual Radar Server
+- dump1090 dump
```
Comment (ItzLevvie) [inline manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.locale.en-US.yaml:23]:
```suggestion
- virtual-radar
- virtual-radar-server
- dump1090
- dump
```

File: manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: vradarserver.vrs
+PackageVersion: 2.4.4
+PackageLocale: en-US
+Publisher: vradarserver
+PublisherUrl: http://www.virtualradarserver.co.uk/Default.aspx
+PublisherSupportUrl: https://forum.virtualradarserver.co.uk
+Author: Andrew Whewell
+PackageName: vrs
```
Comment (ItzLevvie) [inline manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.locale.en-US.yaml:11]:
```suggestion
PackageName: Virtual Radar
```

File: manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: vradarserver.vrs
+PackageVersion: 2.4.4
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: http://www.virtualradarserver.co.uk/Files/VirtualRadarSetup.exe
+  InstallerSha256: 02CB721766B8D2ADEBEB773A168FBAC12619EC275F4C696BF80CB1AFC273E071
```
Comment (ItzLevvie) [inline manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.installer.yaml:10]:
```suggestion
  InstallerSha256: 02CB721766B8D2ADEBEB773A168FBAC12619EC275F4C696BF80CB1AFC273E071
  ProductCode: Virtual Radar_is1
```

File: manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: vradarserver.vrs
+PackageVersion: 2.4.4
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: http://www.virtualradarserver.co.uk/Files/VirtualRadarSetup.exe
```
Comment (ItzLevvie) [inline manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.installer.yaml:9]:
```suggestion
  InstallerUrl: https://www.virtualradarserver.co.uk/Files/VirtualRadarSetup.exe
```

File: manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+PackageUrl: http://www.virtualradarserver.co.uk/Download.aspx
+License: Copyright (C) 2010 onwards, Andrew Whewell All rights reserved.
+LicenseUrl: https://github.com/vradarserver/vrs/blob/master/LICENSE.md
+CopyrightUrl: https://github.com/vradarserver/vrs/blob/master/LICENSE.md
+ShortDescription: Virtual Radar Setup
+Description: '"Virtual Radar Server is an open-source .NET application that runs a local web server"'
```
Comment (ItzLevvie) [inline manifests/v/vradarserver/vrs/2.4.4/vradarserver.vrs.locale.en-US.yaml:17]:
```suggestion
ShortDescription: Virtual Radar Server is an open-source .NET application that runs a local web server.
Description: Virtual Radar Server is an open-source .NET application that runs a local web server.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Boxersteavee** [comment]
I have no idea what's going on here, it worked fine for me during testing.

**Comment by ItzLevvie** [comment]
> I have no idea what's going on here, it worked fine for me during testing.

Wait for an engineer to take a look. The error may also be related to the architecture specified in the manifest (i.e. the installer could be x64 but you listed x86)

**Comment by ChuckFerring** [comment]
The application has a runtime dependency on .Net Framework 3.5.

**Comment by Boxersteavee** [comment]
> > I have no idea what's going on here, it worked fine for me during testing.
> 
> Wait for an engineer to take a look. The error may also be related to the architecture specified in the manifest (i.e. the installer could be x64 but you listed x86)

Website said x86 (altho x64 is available and I was planning on adding that after)

**Comment by Boxersteavee** [comment]
> The application has a runtime dependency on .Net Framework 3.5.

So what do I do then? (sorry for not replying recently I have been super busy IRL)

**Pull request closed without merging**